src/Altinn.Platform/Altinn.Platform.Events/Events/Controllers/EventsController.cs

```diff
@@ -59,5 +69,61 @@ public async Task<ActionResult<string>> Post([FromBody] CloudEvent cloudEvent)
                 return StatusCode(500, $"Unable to store cloud event in database. {e}");
             }
         }
+
+        /// <summary>
+        /// Retrieves a set of events based on query parameters.
+        /// </summary>
+        [HttpGet("{org}/{app}")]
```

Comment:
We need to discuss if we need to add some authorization here before deploying it to production

Reply:
Agreed. The config enabling connection to the database is currently only enabled for YT01, so that's no issue.